dplace_app/models.py

```diff
@@ -350,6 +347,10 @@ class Language(models.Model):
     # needs to be null=True because some glottolog languages do not have isocodes
     iso_code = models.ForeignKey('ISOCode', null=True)
     family = models.ForeignKey('LanguageFamily', null=True)
+    
+    @property
+    def societies_count(self):
+        return Society.objects.all().filter(language=self).count()
```

Comment:
Hm. Doesn't this re-introduce one db query per EnvironmentalVariable which is only run upon serialization?